File: src/pt/up/fe/luisfonseca/cp/api/json/Warnings.java
package pt.up.fe.luisfonseca.cp.api.json;

public class Warnings extends Error {

	public WarningItem[] today;
	public WarningItem[] tomorrow;
	public WarningItem[] after;
	
	public static class WarningItem
	{
		public String title,
					  desc;
	}
}

File: src/pt/up/fe/luisfonseca/cp/api/json/Warnings.scala
package pt.up.fe.luisfonseca.cp.api.json

case class Warnings(
    val today: List[WarningItem],
	val tomorrow: List[WarningItem],
	val after: List[WarningItem],
	val error: String = null, val error_msg:String = null)
	extends Error
{

}

case class WarningItem(val title: String, val desc: String)
{
  
}